conseil-common/src/main/scala/tech/cryptonomic/conseil/common/mantis/MantisTypes.scala

```diff
@@ -0,0 +1,8 @@
+package tech.cryptonomic.conseil.common.mantis
+
+object MantisTypes {
+
+  /** Case class representing hash to identify blocks across many block chains */
+  final case class MantisBlockHash(value: String) extends AnyVal
```

Comment:
For this kind of things i would love to see some `common types` in order to properly model common abstractions between different blockchains.